libnetwork/osl/namespace_linux.go

```diff
@@ -691,6 +680,18 @@ func (n *networkNamespace) ApplyOSTweaks(types []SandboxType) {
 		switch t {
 		case SandboxTypeLoadBalancer:
 			kernel.ApplyOSTweaks(loadBalancerConfig)
+		case SandboxTypeLoadBalancer, SandboxTypeIngress:
+			kernel.ApplyOSTweaks(map[string]*kernel.OSValue{
```

Comment:
This is failing to compile:

```
libnetwork/osl/namespace_linux.go:682:25: undefined: loadBalancerConfig
libnetwork/osl/namespace_linux.go:683:3: duplicate case SandboxTypeLoadBalancer (value 1) in switch
previous case at libnetwork/osl/namespace_linux.go:681:3
```

```suggestion
		case SandboxTypeLoadBalancer, SandboxTypeIngress:
			kernel.ApplyOSTweaks(map[string]*kernel.OSValue{
```

I'm also curious: why move the values inline here instead of retaining the `loadBalancerConfig` package-level variable?